scripts/irods/test/test_irule.py

```diff
@@ -53,3 +54,19 @@ def test_irule_printVariables_on_stdout_4189(self):
         self.assertNotIn( "[1]", stdout )
         self.assertIn( "badInput format error", stderr )
         self.assertNotEqual( rc, 0 )
+
+    @unittest.skipIf(test.settings.TOPOLOGY_FROM_RESOURCE_SERVER or plugin_name=='irods_rule_engine_plugin-python',
+                     'Skip for topology testing from resource server: reads server log')
+    def test_irule_does_not_crash_on_bad_rule_file__issue_7740(self):
+        bad_rule = '''
+        bleh {
+            writeLine("Did I do this right?");
+        }
+        OUTPUT
+        '''
+        path_to_file = os.path.join(self.admin.local_session_dir, 'non.r')
+
+        with open(path_to_file, 'w') as f:
+            f.write(bad_rule)
+
+        self.admin.assert_icommand_fail(['irule', '-F', path_to_file, '-r', 'irods_rule_engine_plugin-irods_rule_language-instance'], desired_rc=1)
```

Comment:
Test is fine/good. Just need to tweak a few words.

Reply:
Which words need tweaking?

Reply:
- bleh
- non.r

Consider replacing them with something involving the issue number. That helps us debug test failures.

Reply:
Renamed.